android/build.gradle

```diff
@@ -35,10 +35,11 @@ def pkgData = new JsonSlurper().parseText(rootPkg.text)
 def sdk_version = pkgData.version
 
 android {
+  namespace = "com.amazonaws.ivs.reactnative.player"
   compileSdkVersion getExtOrIntegerDefault('compileSdkVersion')
   buildToolsVersion getExtOrDefault('buildToolsVersion')
   defaultConfig {
-    minSdkVersion 21
+    minSdkVersion 26
```

Comment:
Looks like we're getting a build issue here:
https://github.com/aws/amazon-ivs-react-native-player/actions/runs/5904085265/job/16025249877?pr=151#step:11:817